plugins/fzf/fzf.plugin.zsh

```diff
@@ -173,6 +173,32 @@ function fzf_setup_using_cygwin() {
   return 0
 }
 
+function fzf_setup_using_macports() {
+  # If the command is not found, the package isn't installed
+  (( $+commands[fzf] )) || return 1
+
+  # The fzf-zsh-completion package installs the auto-completion in
```

Comment:
What package is this? It doesn't seem to be from macports.

The following  path also seems at odds with the information in the fzf macport: https://ports.macports.org/port/fzf/details/. Quoting from it:

```
Append this line to ~/.zshrc to enable fuzzy auto-completion for Zsh:

source ${prefix}/share/fzf/shell/completion.zsh
```

On my system, with macports fzf installed, the path `/opt/local/share/zsh/site-functions/fzf` does not exist.

Reply:
Let me investigate something and get back to you later today

Reply:
Any news?

Reply:
This PR is referring to an older version of the macport: https://github.com/macports/macports-ports/commit/b90c2574a1b62cabc202848d2b8684d771cdbd24. We can update the plugin and let people know how to update the macport.